CaloDigi/Realistic/doc/Readme_ROCTiming.md

```diff
@@ -0,0 +1,57 @@
+
+### Processor parameters
+
+Here are listed the processor parameters acting on timing.
+
+General parameters:
+
+- `timingWindowMin`: The minimum hit time to accept a hit (t > `t_min`). Float, unit in `ns`, default is -10.
+- `timingWindowMax`: The maximum hit time to accept a hit (t < `t_max`). Float, unit in `ns`, default is 100.
+- `integrationMethod`: The energy integration and timing estimate method for digitization. Options are "Standard" (default) and "ROC".
+
+Parameters for the "Standard" method **only**:
+
+- `timingCorrectForPropagation`: Hit time correction `t_corr` for propagation computed as radial distance divided by `c`. Hit time becomes t - t_corr. Int, default is 0 (false).
+
+Parameters for the "ROC" method **only**:
+
+- `fastShaper`: The fast shaper integration time `t_fast`. Must be set in the steering file if the ROC method is chosen. Float, unit in `ns`, default is 0.
+- `slowShaper`: The slow shaper integration time `t_slow`. Must be set in the steering file if the ROC method is chosen. Float, unit in `ns`, default is 0.
+- `timingResolution`: Apply a random gaussian smearing `t_res` on the final hit time. The parameter value corresponds to the width of the gaussian function. Only applied if greater than 0. Float, unit in `ns`, default is 0 (no time smearing).
+
+The default behavior of the digitizer is then:
+
+- `integrationMethod`: "Standard"
+- `timingWindowMin`: -10 ns
+- `timingWindowMax`: 100 ns
+- `timingCorrectForPropagation`: false
+
+
+### Additional remarks on the ROC implementation
+
+#### Pull request
+The ROC implementation has been implementation through the `PR83` in the MarlinReco Github repository: https://github.com/iLCSoft/MarlinReco/pull/83. More detailed information, in particular the discussion with detector experts, can be found there.
+
+#### Current ROC parameter values for known technologies (03.2021)
+
+- AHCal:
+  - `fastShaper`: 15 ns
+  - `slowShaper`: 50 ns
+  - `timingResolution`: 700 ps
+- SiWEcal:
+  - `fastShaper`: 90 ns. First estimate
+  - `slowShaper`: 180 ns. First estimate
+  - `timingResolution`: 700 ps. First guess from AHCAL value, as the TDC is similar. Not a real estimate
```

Comment:
Can we get a number of the SiW ECAL prototype? I would think it is in the same ballpark or even better.

Reply:
@airqui ping